### ISSUE TITLE
Package components as ES6 classes 

### ISSUE THREAD

**Issue by ZhangYiJiang** [miukimiu/react-kawaii#45]
Currently because of the way this library is packaged as UMD, importing any component from the library will cause the all components to be imported. It would be great if the components can also be built as ES6 and made available under the `modules` entry point in `package.json` so that ES6 aware bundlers can shake away unused components.

**Comment by miukimiu** [comment]
Hello @ZhangYiJiang,

Do you know how to do that? Do you want to do a PR?

Thanks!

**Comment by ZhangYiJiang** [comment]
I've not done it before, but I've seen it done using [Rollup](https://github.com/rollup/rollup/). I don't have time this week, unfortunately, but I can take a look maybe this or next weekend

**Comment by ZhangYiJiang** [comment]
I gave this a try yesterday. A few things I've found 

- This package uses Webpack, but only Rollup supports ES6 export target. Webpack 5 may support ES6 as a target, but this package is only on Webpack 2. This means either switching to Rollup entirely, or maintaining two different build systems, both of which are annoying options 
- While the results from using Rollup looks correct, when imported Webpack still does not appear to do tree shaking, ie. the entire file is included instead of just the components used. This is very annoying, but I can try investigating further
